phoronix/run_phoronix.sh

```diff
@@ -256,12 +256,22 @@ else
 	${curdir}/test_tools/move_data $curdir  results_${test_name}_${to_tuned_setting}/phoronix_results/results_phoronix
 	cp /tmp/results_${test_name}_${to_tuned_setting}.out results_${test_name}_${to_tuned_setting}/phoronix_results/results_phoronix
 	pushd /tmp/results_${test_name}_${to_tuned_setting}/phoronix_results/results_phoronix > /dev/null
-	$run_dir/reduce_phoronix > results.csv
-	lines=`wc -l results.csv | cut -d' ' -f 1`
+	$TOOLS_BIN/test_header_info --front_matter --results_file results.csv --host $to_configuration --sys_type $to_sys_type --tuned $to_tuned_setting --results_version $GIT_VERSION --test_name $test_name
+	#
+	# We place the results first in results_check.csv so we can check to make sure
+	# the tests actually ran.  After the check, we will add the run info to results.csv.
+	#
+	$run_dir/reduce_phoronix > results_check.csv
+	lines=`wc -l results_check.csv | cut -d' ' -f 1`
 	if [[ $lines == "1" ]]; then
+		#
+		# We failed, report and do not remove the results_check.csv file.
+		#
 		echo Failed >> test_results_report
 	else
 		echo Ran >> test_results_report
```

Comment:
I'm left thinking this is a missed opportunity.  The script always finishes with 'exit 0' which means that ansible will think it succeeded when in fact we're saying here that it failed.  That seems like a missed opportunity.